03-31 16:31:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO USERS VALUES ('John', 'Doe', 'xxxx', 27)
INSERT[*] INTO USERS VALUES ('Jane', 'Doe', 'yyyy', 28)
INSERT INTO USERS VALUES ('Bill', 'Flood', 'aaaa', 29)
INSERT INTO USERS VALUES ('Veb', 'Nordhagen', 'bbbb', 30)
INSERT INTO USERS VALUES ('Hazel', 'Nutting', 'cccc', 4)
INSERT INTO USERS VALUES ('Caleb', 'Manu', 'dddd', 7)
INSERT INTO USERS VALUES ('Aiden', 'Hall', 'eeee', 19) "; SQL statement:
insert into users values ('John', 'Doe', 'xxxx', 27)
insert into users values ('Jane', 'Doe', 'yyyy', 28)
insert into users values ('Bill', 'Flood', 'aaaa', 29)
insert into users values ('Veb', 'Nordhagen', 'bbbb', 30)
insert into users values ('Hazel', 'Nutting', 'cccc', 4)
insert into users values ('Caleb', 'Manu', 'dddd', 7)
insert into users values ('Aiden', 'Hall', 'eeee', 19) [42000-186]
03-31 16:53:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Index "PRIMARY_KEY_" not found; SQL statement:
create table lives (
  uid int,
  aid int,
  foreign key (uid) references users,
  foreign key (aid) references address,
  unique(uid,aid)
) [42112-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.table.Table.getPrimaryKey(Table.java:715)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:217)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:171)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:745)
03-31 16:54:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE USERS (
  ID BIGINT,
  FNAME VARCHAR(25),
  LNAME VARCHAR(25),
  PASSWORD VARCHAR(15),
  AGE INT
  PRIMARY KEY ([*]UID)
) "; expected "HASH, AUTO_INCREMENT, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
create table users (
  id bigint,
  fname varchar(25),
  lname varchar(25),
  password varchar(15),
  age int
  primary key (uid)
) [42001-186]
03-31 16:54:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE USERS (
  UID BIGINT,
  FNAME VARCHAR(25),
  LNAME VARCHAR(25),
  PASSWORD VARCHAR(15),
  AGE INT
  PRIMARY KEY ([*]UID)
) "; expected "HASH, AUTO_INCREMENT, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
create table users (
  uid bigint,
  fname varchar(25),
  lname varchar(25),
  password varchar(15),
  age int
  primary key (uid)
) [42001-186]
03-31 16:57:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
insert into address values (1, '1 mallard drive', 'cambridge', 'MA', '34567') [21002-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:247)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:745)
03-31 16:57:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
insert into address values (2, '21 jump street', 'new york', 'NY', '98765') [21002-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:247)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:745)
03-31 16:57:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
insert into address values (3, '4 cherry lane', 'truffala', 'NJ', '58235') [21002-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:247)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:745)
03-31 16:57:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
insert into address values (4, '16 strong road', 'chutney', 'VT', '38573') [21002-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:247)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:745)
03-31 16:57:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
insert into address values (5, '99 livingston circle', 'amherst', 'MA', '99822') [21002-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:247)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:745)
03-31 16:57:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
insert into address values (6, '1123 main street', 'worcester', 'MA', '22234') [21002-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:247)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Thread.java:745)
04-01 12:17:51 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:354)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2785)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2386)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:75)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1148)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at scalikejdbc.CommonsConnectionPool.borrow(CommonsConnectionPool.scala:43)
	at scalikejdbc.DB$.readOnly(DB.scala:172)
	at scalikejdbc.SQLToResult$class.apply(SQL.scala:539)
	at scalikejdbc.SQLToListImpl.apply(SQL.scala:651)
	at cs220.users.User$.list(User.scala:37)
	at $line4.$read$$iw$$iw$$iw$$iw$.<init>(<console>:11)
	at $line4.$read$$iw$$iw$$iw$$iw$.<clinit>(<console>)
	at $line4.$eval$.$print$lzycompute(<console>:7)
	at $line4.$eval$.$print(<console>:6)
	at $line4.$eval.$print(<console>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at scala.tools.nsc.interpreter.IMain$ReadEvalPrint.call(IMain.scala:773)
	at scala.tools.nsc.interpreter.IMain$Request.loadAndRun(IMain.scala:1020)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:627)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:626)
	at scala.reflect.internal.util.ScalaClassLoader$class.asContext(ScalaClassLoader.scala:31)
	at scala.reflect.internal.util.AbstractFileClassLoader.asContext(AbstractFileClassLoader.scala:19)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest.loadAndRunReq(IMain.scala:626)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:558)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:554)
	at scala.tools.nsc.interpreter.ILoop.reallyInterpret$1(ILoop.scala:804)
	at scala.tools.nsc.interpreter.ILoop.interpretStartingWith(ILoop.scala:849)
	at scala.tools.nsc.interpreter.ILoop.command(ILoop.scala:707)
	at scala.tools.nsc.interpreter.ILoop.processLine(ILoop.scala:405)
	at scala.tools.nsc.interpreter.ILoop.loop(ILoop.scala:431)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply$mcZ$sp(ILoop.scala:920)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.reflect.internal.util.ScalaClassLoader$.savingContextLoader(ScalaClassLoader.scala:97)
	at scala.tools.nsc.interpreter.ILoop.process(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop.main(ILoop.scala:931)
	at xsbt.ConsoleInterface.run(ConsoleInterface.scala:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at sbt.compiler.AnalyzingCompiler.call(AnalyzingCompiler.scala:101)
	at sbt.compiler.AnalyzingCompiler.console(AnalyzingCompiler.scala:76)
	at sbt.Console.sbt$Console$$console0$1(Console.scala:22)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply$mcV$sp(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Logger$$anon$4.apply(Logger.scala:85)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 75 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:165)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:340)
	... 70 more
Caused by: java.nio.channels.OverlappingFileLockException
	at sun.nio.ch.SharedFileLockTable.checkList(FileLockTable.java:255)
	at sun.nio.ch.SharedFileLockTable.add(FileLockTable.java:152)
	at sun.nio.ch.FileChannelImpl.tryLock(FileChannelImpl.java:1075)
	at org.h2.store.fs.FileNio.tryLock(FilePathNio.java:121)
	at org.h2.mvstore.cache.FilePathCache$FileCache.tryLock(FilePathCache.java:152)
	at java.nio.channels.FileChannel.tryLock(FileChannel.java:1155)
	at org.h2.mvstore.FileStore.open(FileStore.java:162)
	... 71 more
04-01 12:22:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
insert into users values (?, ?, ?) [21002-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:247)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:72)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:277)
	at org.apache.commons.dbcp.DelegatingConnection.prepareStatement(DelegatingConnection.java:281)
	at org.apache.commons.dbcp.PoolingDataSource$PoolGuardConnectionWrapper.prepareStatement(PoolingDataSource.java:313)
	at scalikejdbc.DBSession$class.createStatementExecutor(DBSession.scala:82)
	at scalikejdbc.DBSession$class.updateWithFilters(DBSession.scala:390)
	at scalikejdbc.ActiveSession.updateWithFilters(DBSession.scala:547)
	at scalikejdbc.DBSession$class.updateWithFilters(DBSession.scala:372)
	at scalikejdbc.ActiveSession.updateWithFilters(DBSession.scala:547)
	at scalikejdbc.SQLUpdate$$anonfun$6.apply(SQL.scala:492)
	at scalikejdbc.SQLUpdate$$anonfun$6.apply(SQL.scala:492)
	at scalikejdbc.DBConnection$class.autoCommit(DBConnection.scala:222)
	at scalikejdbc.DB.autoCommit(DB.scala:60)
	at scalikejdbc.DB$$anonfun$autoCommit$1.apply(DB.scala:215)
	at scalikejdbc.DB$$anonfun$autoCommit$1.apply(DB.scala:214)
	at scalikejdbc.LoanPattern$class.using(LoanPattern.scala:18)
	at scalikejdbc.DB$.using(DB.scala:138)
	at scalikejdbc.DB$.autoCommit(DB.scala:214)
	at scalikejdbc.SQLUpdate.apply(SQL.scala:492)
	at cs220.users.User$.make(User.scala:57)
	at $line9.$read$$iw$$iw$$iw$$iw$.<init>(<console>:11)
	at $line9.$read$$iw$$iw$$iw$$iw$.<clinit>(<console>)
	at $line9.$eval$.$print$lzycompute(<console>:7)
	at $line9.$eval$.$print(<console>:6)
	at $line9.$eval.$print(<console>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at scala.tools.nsc.interpreter.IMain$ReadEvalPrint.call(IMain.scala:773)
	at scala.tools.nsc.interpreter.IMain$Request.loadAndRun(IMain.scala:1020)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:627)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:626)
	at scala.reflect.internal.util.ScalaClassLoader$class.asContext(ScalaClassLoader.scala:31)
	at scala.reflect.internal.util.AbstractFileClassLoader.asContext(AbstractFileClassLoader.scala:19)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest.loadAndRunReq(IMain.scala:626)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:558)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:554)
	at scala.tools.nsc.interpreter.ILoop.reallyInterpret$1(ILoop.scala:804)
	at scala.tools.nsc.interpreter.ILoop.interpretStartingWith(ILoop.scala:849)
	at scala.tools.nsc.interpreter.ILoop.command(ILoop.scala:707)
	at scala.tools.nsc.interpreter.ILoop.processLine(ILoop.scala:405)
	at scala.tools.nsc.interpreter.ILoop.loop(ILoop.scala:431)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply$mcZ$sp(ILoop.scala:920)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.reflect.internal.util.ScalaClassLoader$.savingContextLoader(ScalaClassLoader.scala:97)
	at scala.tools.nsc.interpreter.ILoop.process(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop.main(ILoop.scala:931)
	at xsbt.ConsoleInterface.run(ConsoleInterface.scala:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at sbt.compiler.AnalyzingCompiler.call(AnalyzingCompiler.scala:101)
	at sbt.compiler.AnalyzingCompiler.console(AnalyzingCompiler.scala:76)
	at sbt.Console.sbt$Console$$console0$1(Console.scala:22)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply$mcV$sp(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Logger$$anon$4.apply(Logger.scala:85)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:745)
04-01 12:33:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:


insert into users values ('Aiden', 'Hall', 'eeee', 19) [21002-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepare(Parser.java:227)
	at org.h2.engine.Session.prepare(Session.java:412)
	at org.h2.engine.Session.prepare(Session.java:399)
	at org.h2.command.dml.RunScriptCommand.execute(RunScriptCommand.java:74)
	at org.h2.command.dml.RunScriptCommand.update(RunScriptCommand.java:57)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.tools.Shell.execute(Shell.java:454)
	at org.h2.tools.Shell.promptLoop(Shell.java:293)
	at org.h2.tools.Shell.runTool(Shell.java:150)
	at org.h2.tools.Shell.main(Shell.java:79)
04-01 12:34:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:


insert into address values ('1 mallard drive', 'cambridge', 'MA', '34567') [21002-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepare(Parser.java:227)
	at org.h2.engine.Session.prepare(Session.java:412)
	at org.h2.engine.Session.prepare(Session.java:399)
	at org.h2.command.dml.RunScriptCommand.execute(RunScriptCommand.java:74)
	at org.h2.command.dml.RunScriptCommand.update(RunScriptCommand.java:57)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.tools.Shell.execute(Shell.java:454)
	at org.h2.tools.Shell.promptLoop(Shell.java:293)
	at org.h2.tools.Shell.runTool(Shell.java:150)
	at org.h2.tools.Shell.main(Shell.java:79)
04-01 12:35:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
insert into address values ('1 mallard drive', 'cambridge', 'MA', '34567') [21002-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:247)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.tools.Shell.execute(Shell.java:454)
	at org.h2.tools.Shell.promptLoop(Shell.java:293)
	at org.h2.tools.Shell.runTool(Shell.java:150)
	at org.h2.tools.Shell.main(Shell.java:79)
04-01 12:35:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "

INSERT INTO ADDRESS (STREET, CITY, STATE, ZIPCODE) VALUES
  ('1 mallard drive', 'cambridge', 'MA', '34567')
  ([*]'21 jump street', 'new york', 'NY', '98765'),
  ('4 cherry lane', 'truffala', 'NJ', '58235'),
  ('16 strong road', 'chutney', 'VT', '38573'),
  ('99 livingston circle', 'amherst', 'MA', '99822'),
  ('1123 main street', 'worcester', 'MA', '22234') "; SQL statement:


insert into address (street, city, state, zipcode) values
  ('1 mallard drive', 'cambridge', 'MA', '34567')
  ('21 jump street', 'new york', 'NY', '98765'),
  ('4 cherry lane', 'truffala', 'NJ', '58235'),
  ('16 strong road', 'chutney', 'VT', '38573'),
  ('99 livingston circle', 'amherst', 'MA', '99822'),
  ('1123 main street', 'worcester', 'MA', '22234') [42000-186]
04-01 12:35:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "  ('1 mallard drive'[*], 'cambridge', 'MA', '34567')
"; expected "(, SELECT, FROM"; SQL statement:
  ('1 mallard drive', 'cambridge', 'MA', '34567')
 [42001-186]
04-01 12:35:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:


insert into lives(uid, aid)
  select fname, street, uid, aid
  from   users, address
  where  street = '1123 main street'
    and  fname  = 'Aiden'

-- insert into lives values
--   (1, 6),
--   (2, 4),
--   (3, 6),
--   (4, 2),
--   (5, 5),
--   (6, 1),
--   (7, 3); [21002-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:282)
	at org.h2.command.Parser.prepare(Parser.java:227)
	at org.h2.engine.Session.prepare(Session.java:412)
	at org.h2.engine.Session.prepare(Session.java:399)
	at org.h2.command.dml.RunScriptCommand.execute(RunScriptCommand.java:74)
	at org.h2.command.dml.RunScriptCommand.update(RunScriptCommand.java:57)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.tools.Shell.execute(Shell.java:454)
	at org.h2.tools.Shell.promptLoop(Shell.java:293)
	at org.h2.tools.Shell.runTool(Shell.java:150)
	at org.h2.tools.Shell.main(Shell.java:79)
04-01 12:40:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "

INSERT INTO LIVES(UID, AID)
  SELECT USERS.UID, ADDRESS.AID
  FROM   USERS, ADDRESS
  WHERE  STREET = '1123 main street'
    AND  FNAME  = 'Aiden'

INSERT[*] INTO LIVES(UID, AID)
  SELECT USERS.UID, ADDRESS.AID
  FROM   USERS, ADDRESS
  WHERE  STREET = '16 strong road'
    AND  FNAME  = 'Caleb'

INSERT INTO LIVES(UID, AID)
  SELECT USERS.UID, ADDRESS.AID
  FROM   USERS, ADDRESS
  WHERE  STREET = '1123 main street'
    AND  FNAME  = 'Hazel'

INSERT INTO LIVES(UID, AID)
  SELECT USERS.UID, ADDRESS.AID
  FROM   USERS, ADDRESS
  WHERE  STREET = '21 jump street'
    AND  FNAME  = 'Veb'

INSERT INTO LIVES(UID, AID)
  SELECT USERS.UID, ADDRESS.AID
  FROM   USERS, ADDRESS
  WHERE  STREET = '99 livingston circle'
    AND  FNAME  = 'Bill'

INSERT INTO LIVES(UID, AID)
  SELECT USERS.UID, ADDRESS.AID
  FROM   USERS, ADDRESS
  WHERE  STREET = '1 mallard drive'
    AND  FNAME  = 'Jane'

INSERT INTO LIVES(UID, AID)
  SELECT USERS.UID, ADDRESS.AID
  FROM   USERS, ADDRESS
  WHERE  STREET = '4 cherry lane'
    AND  FNAME  = 'John'




                           
           ; "; SQL statement:


insert into lives(uid, aid)
  select users.uid, address.aid
  from   users, address
  where  street = '1123 main street'
    and  fname  = 'Aiden'

insert into lives(uid, aid)
  select users.uid, address.aid
  from   users, address
  where  street = '16 strong road'
    and  fname  = 'Caleb'

insert into lives(uid, aid)
  select users.uid, address.aid
  from   users, address
  where  street = '1123 main street'
    and  fname  = 'Hazel'

insert into lives(uid, aid)
  select users.uid, address.aid
  from   users, address
  where  street = '21 jump street'
    and  fname  = 'Veb'

insert into lives(uid, aid)
  select users.uid, address.aid
  from   users, address
  where  street = '99 livingston circle'
    and  fname  = 'Bill'

insert into lives(uid, aid)
  select users.uid, address.aid
  from   users, address
  where  street = '1 mallard drive'
    and  fname  = 'Jane'

insert into lives(uid, aid)
  select users.uid, address.aid
  from   users, address
  where  street = '4 cherry lane'
    and  fname  = 'John'




-- insert into lives values
--   (7, 3); [42000-186]
04-02 13:30:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
insert into users values (?, ?, ?) [21002-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:247)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:72)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:277)
	at org.apache.commons.dbcp.DelegatingConnection.prepareStatement(DelegatingConnection.java:281)
	at org.apache.commons.dbcp.PoolingDataSource$PoolGuardConnectionWrapper.prepareStatement(PoolingDataSource.java:313)
	at scalikejdbc.DBSession$class.createStatementExecutor(DBSession.scala:82)
	at scalikejdbc.DBSession$class.updateWithFilters(DBSession.scala:390)
	at scalikejdbc.ActiveSession.updateWithFilters(DBSession.scala:547)
	at scalikejdbc.DBSession$class.updateWithFilters(DBSession.scala:372)
	at scalikejdbc.ActiveSession.updateWithFilters(DBSession.scala:547)
	at scalikejdbc.SQLUpdate$$anonfun$6.apply(SQL.scala:492)
	at scalikejdbc.SQLUpdate$$anonfun$6.apply(SQL.scala:492)
	at scalikejdbc.DBConnection$class.autoCommit(DBConnection.scala:222)
	at scalikejdbc.DB.autoCommit(DB.scala:60)
	at scalikejdbc.DB$$anonfun$autoCommit$1.apply(DB.scala:215)
	at scalikejdbc.DB$$anonfun$autoCommit$1.apply(DB.scala:214)
	at scalikejdbc.LoanPattern$class.using(LoanPattern.scala:18)
	at scalikejdbc.DB$.using(DB.scala:138)
	at scalikejdbc.DB$.autoCommit(DB.scala:214)
	at scalikejdbc.SQLUpdate.apply(SQL.scala:492)
	at cs220.users.User$.make(User.scala:60)
	at $line7.$read$$iw$$iw$$iw$$iw$.<init>(<console>:11)
	at $line7.$read$$iw$$iw$$iw$$iw$.<clinit>(<console>)
	at $line7.$eval$.$print$lzycompute(<console>:7)
	at $line7.$eval$.$print(<console>:6)
	at $line7.$eval.$print(<console>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at scala.tools.nsc.interpreter.IMain$ReadEvalPrint.call(IMain.scala:773)
	at scala.tools.nsc.interpreter.IMain$Request.loadAndRun(IMain.scala:1020)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:627)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:626)
	at scala.reflect.internal.util.ScalaClassLoader$class.asContext(ScalaClassLoader.scala:31)
	at scala.reflect.internal.util.AbstractFileClassLoader.asContext(AbstractFileClassLoader.scala:19)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest.loadAndRunReq(IMain.scala:626)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:558)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:554)
	at scala.tools.nsc.interpreter.ILoop.reallyInterpret$1(ILoop.scala:804)
	at scala.tools.nsc.interpreter.ILoop.interpretStartingWith(ILoop.scala:849)
	at scala.tools.nsc.interpreter.ILoop.command(ILoop.scala:707)
	at scala.tools.nsc.interpreter.ILoop.processLine(ILoop.scala:405)
	at scala.tools.nsc.interpreter.ILoop.loop(ILoop.scala:431)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply$mcZ$sp(ILoop.scala:920)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.reflect.internal.util.ScalaClassLoader$.savingContextLoader(ScalaClassLoader.scala:97)
	at scala.tools.nsc.interpreter.ILoop.process(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop.main(ILoop.scala:931)
	at xsbt.ConsoleInterface.run(ConsoleInterface.scala:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at sbt.compiler.AnalyzingCompiler.call(AnalyzingCompiler.scala:101)
	at sbt.compiler.AnalyzingCompiler.console(AnalyzingCompiler.scala:76)
	at sbt.Console.sbt$Console$$console0$1(Console.scala:22)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply$mcV$sp(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Logger$$anon$4.apply(Logger.scala:85)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:745)
04-02 13:32:49 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:354)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2785)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2386)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:75)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1148)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at scalikejdbc.CommonsConnectionPool.borrow(CommonsConnectionPool.scala:43)
	at scalikejdbc.DB$.autoCommit(DB.scala:214)
	at scalikejdbc.SQLUpdate.apply(SQL.scala:492)
	at cs220.users.User$.make(User.scala:63)
	at $line4.$read$$iw$$iw$$iw$$iw$.<init>(<console>:11)
	at $line4.$read$$iw$$iw$$iw$$iw$.<clinit>(<console>)
	at $line4.$eval$.$print$lzycompute(<console>:7)
	at $line4.$eval$.$print(<console>:6)
	at $line4.$eval.$print(<console>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at scala.tools.nsc.interpreter.IMain$ReadEvalPrint.call(IMain.scala:773)
	at scala.tools.nsc.interpreter.IMain$Request.loadAndRun(IMain.scala:1020)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:627)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:626)
	at scala.reflect.internal.util.ScalaClassLoader$class.asContext(ScalaClassLoader.scala:31)
	at scala.reflect.internal.util.AbstractFileClassLoader.asContext(AbstractFileClassLoader.scala:19)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest.loadAndRunReq(IMain.scala:626)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:558)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:554)
	at scala.tools.nsc.interpreter.ILoop.reallyInterpret$1(ILoop.scala:804)
	at scala.tools.nsc.interpreter.ILoop.interpretStartingWith(ILoop.scala:849)
	at scala.tools.nsc.interpreter.ILoop.command(ILoop.scala:707)
	at scala.tools.nsc.interpreter.ILoop.processLine(ILoop.scala:405)
	at scala.tools.nsc.interpreter.ILoop.loop(ILoop.scala:431)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply$mcZ$sp(ILoop.scala:920)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.reflect.internal.util.ScalaClassLoader$.savingContextLoader(ScalaClassLoader.scala:97)
	at scala.tools.nsc.interpreter.ILoop.process(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop.main(ILoop.scala:931)
	at xsbt.ConsoleInterface.run(ConsoleInterface.scala:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at sbt.compiler.AnalyzingCompiler.call(AnalyzingCompiler.scala:101)
	at sbt.compiler.AnalyzingCompiler.console(AnalyzingCompiler.scala:76)
	at sbt.Console.sbt$Console$$console0$1(Console.scala:22)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply$mcV$sp(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Logger$$anon$4.apply(Logger.scala:85)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 74 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:165)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:340)
	... 69 more
Caused by: java.nio.channels.OverlappingFileLockException
	at sun.nio.ch.SharedFileLockTable.checkList(FileLockTable.java:255)
	at sun.nio.ch.SharedFileLockTable.add(FileLockTable.java:152)
	at sun.nio.ch.FileChannelImpl.tryLock(FileChannelImpl.java:1075)
	at org.h2.store.fs.FileNio.tryLock(FilePathNio.java:121)
	at org.h2.mvstore.cache.FilePathCache$FileCache.tryLock(FilePathCache.java:152)
	at java.nio.channels.FileChannel.tryLock(FileChannel.java:1155)
	at org.h2.mvstore.FileStore.open(FileStore.java:162)
	... 70 more
04-02 13:33:38 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:354)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2785)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2386)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:75)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1148)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at scalikejdbc.CommonsConnectionPool.borrow(CommonsConnectionPool.scala:43)
	at scalikejdbc.DB$.readOnly(DB.scala:172)
	at scalikejdbc.SQLToResult$class.apply(SQL.scala:539)
	at scalikejdbc.SQLToListImpl.apply(SQL.scala:651)
	at cs220.users.User$.list(User.scala:44)
	at $line4.$read$$iw$$iw$$iw$$iw$.<init>(<console>:11)
	at $line4.$read$$iw$$iw$$iw$$iw$.<clinit>(<console>)
	at $line4.$eval$.$print$lzycompute(<console>:7)
	at $line4.$eval$.$print(<console>:6)
	at $line4.$eval.$print(<console>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at scala.tools.nsc.interpreter.IMain$ReadEvalPrint.call(IMain.scala:773)
	at scala.tools.nsc.interpreter.IMain$Request.loadAndRun(IMain.scala:1020)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:627)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:626)
	at scala.reflect.internal.util.ScalaClassLoader$class.asContext(ScalaClassLoader.scala:31)
	at scala.reflect.internal.util.AbstractFileClassLoader.asContext(AbstractFileClassLoader.scala:19)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest.loadAndRunReq(IMain.scala:626)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:558)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:554)
	at scala.tools.nsc.interpreter.ILoop.reallyInterpret$1(ILoop.scala:804)
	at scala.tools.nsc.interpreter.ILoop.interpretStartingWith(ILoop.scala:849)
	at scala.tools.nsc.interpreter.ILoop.command(ILoop.scala:707)
	at scala.tools.nsc.interpreter.ILoop.processLine(ILoop.scala:405)
	at scala.tools.nsc.interpreter.ILoop.loop(ILoop.scala:431)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply$mcZ$sp(ILoop.scala:920)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.reflect.internal.util.ScalaClassLoader$.savingContextLoader(ScalaClassLoader.scala:97)
	at scala.tools.nsc.interpreter.ILoop.process(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop.main(ILoop.scala:931)
	at xsbt.ConsoleInterface.run(ConsoleInterface.scala:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at sbt.compiler.AnalyzingCompiler.call(AnalyzingCompiler.scala:101)
	at sbt.compiler.AnalyzingCompiler.console(AnalyzingCompiler.scala:76)
	at sbt.Console.sbt$Console$$console0$1(Console.scala:22)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply$mcV$sp(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Logger$$anon$4.apply(Logger.scala:85)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 75 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:165)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:340)
	... 70 more
Caused by: java.nio.channels.OverlappingFileLockException
	at sun.nio.ch.SharedFileLockTable.checkList(FileLockTable.java:255)
	at sun.nio.ch.SharedFileLockTable.add(FileLockTable.java:152)
	at sun.nio.ch.FileChannelImpl.tryLock(FileChannelImpl.java:1075)
	at org.h2.store.fs.FileNio.tryLock(FilePathNio.java:121)
	at org.h2.mvstore.cache.FilePathCache$FileCache.tryLock(FilePathCache.java:152)
	at java.nio.channels.FileChannel.tryLock(FileChannel.java:1155)
	at org.h2.mvstore.FileStore.open(FileStore.java:162)
	... 71 more
04-02 13:33:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
insert into users values (?, ?, ?, ?) [21002-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:247)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:72)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:277)
	at org.apache.commons.dbcp.DelegatingConnection.prepareStatement(DelegatingConnection.java:281)
	at org.apache.commons.dbcp.PoolingDataSource$PoolGuardConnectionWrapper.prepareStatement(PoolingDataSource.java:313)
	at scalikejdbc.DBSession$class.createStatementExecutor(DBSession.scala:82)
	at scalikejdbc.DBSession$class.updateWithFilters(DBSession.scala:390)
	at scalikejdbc.ActiveSession.updateWithFilters(DBSession.scala:547)
	at scalikejdbc.DBSession$class.updateWithFilters(DBSession.scala:372)
	at scalikejdbc.ActiveSession.updateWithFilters(DBSession.scala:547)
	at scalikejdbc.SQLUpdate$$anonfun$6.apply(SQL.scala:492)
	at scalikejdbc.SQLUpdate$$anonfun$6.apply(SQL.scala:492)
	at scalikejdbc.DBConnection$class.autoCommit(DBConnection.scala:222)
	at scalikejdbc.DB.autoCommit(DB.scala:60)
	at scalikejdbc.DB$$anonfun$autoCommit$1.apply(DB.scala:215)
	at scalikejdbc.DB$$anonfun$autoCommit$1.apply(DB.scala:214)
	at scalikejdbc.LoanPattern$class.using(LoanPattern.scala:18)
	at scalikejdbc.DB$.using(DB.scala:138)
	at scalikejdbc.DB$.autoCommit(DB.scala:214)
	at scalikejdbc.SQLUpdate.apply(SQL.scala:492)
	at cs220.users.User$.make(User.scala:63)
	at $line4.$read$$iw$$iw$$iw$$iw$.<init>(<console>:11)
	at $line4.$read$$iw$$iw$$iw$$iw$.<clinit>(<console>)
	at $line4.$eval$.$print$lzycompute(<console>:7)
	at $line4.$eval$.$print(<console>:6)
	at $line4.$eval.$print(<console>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at scala.tools.nsc.interpreter.IMain$ReadEvalPrint.call(IMain.scala:773)
	at scala.tools.nsc.interpreter.IMain$Request.loadAndRun(IMain.scala:1020)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:627)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:626)
	at scala.reflect.internal.util.ScalaClassLoader$class.asContext(ScalaClassLoader.scala:31)
	at scala.reflect.internal.util.AbstractFileClassLoader.asContext(AbstractFileClassLoader.scala:19)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest.loadAndRunReq(IMain.scala:626)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:558)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:554)
	at scala.tools.nsc.interpreter.ILoop.reallyInterpret$1(ILoop.scala:804)
	at scala.tools.nsc.interpreter.ILoop.interpretStartingWith(ILoop.scala:849)
	at scala.tools.nsc.interpreter.ILoop.command(ILoop.scala:707)
	at scala.tools.nsc.interpreter.ILoop.processLine(ILoop.scala:405)
	at scala.tools.nsc.interpreter.ILoop.loop(ILoop.scala:431)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply$mcZ$sp(ILoop.scala:920)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.reflect.internal.util.ScalaClassLoader$.savingContextLoader(ScalaClassLoader.scala:97)
	at scala.tools.nsc.interpreter.ILoop.process(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop.main(ILoop.scala:931)
	at xsbt.ConsoleInterface.run(ConsoleInterface.scala:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at sbt.compiler.AnalyzingCompiler.call(AnalyzingCompiler.scala:101)
	at sbt.compiler.AnalyzingCompiler.console(AnalyzingCompiler.scala:76)
	at sbt.Console.sbt$Console$$console0$1(Console.scala:22)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply$mcV$sp(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Logger$$anon$4.apply(Logger.scala:85)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:745)
04-02 14:04:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "
      select *
      from   address
      where  street=?
        and  city=?
        and  state={?[*]}
        and  zipcode=?
    "; expected "=" [42001-186]
04-02 14:06:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "
      select *
      from   address
      where  street=?
        and  city=?
        and  state={?[*]}
        and  zipcode=?
    "; expected "=" [42001-186]
04-02 14:07:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "
      select *
      from   address
      where  street = ?
        and  city = ?
        and  state = {?[*]}
        and  zipcode = ?
    "; expected "=" [42001-186]
04-02 14:09:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "
      select *
      from   address
      where  street=?
        and  city=?
        and  state={?[*]}
    "; expected "=" [42001-186]
04-02 14:12:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "
      select *
      from   address
      where  street = ?
        and  city = ?
        and  state = {?[*]}
    "; expected "=" [42001-186]
04-02 14:30:28 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:354)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2785)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2386)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:75)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1148)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at scalikejdbc.CommonsConnectionPool.borrow(CommonsConnectionPool.scala:43)
	at scalikejdbc.DB$.readOnly(DB.scala:172)
	at scalikejdbc.SQLToResult$class.apply(SQL.scala:539)
	at scalikejdbc.SQLToListImpl.apply(SQL.scala:651)
	at cs220.users.Address$.list(Address.scala:64)
	at $line5.$read$$iw$$iw$$iw$$iw$.<init>(<console>:11)
	at $line5.$read$$iw$$iw$$iw$$iw$.<clinit>(<console>)
	at $line5.$eval$.$print$lzycompute(<console>:7)
	at $line5.$eval$.$print(<console>:6)
	at $line5.$eval.$print(<console>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at scala.tools.nsc.interpreter.IMain$ReadEvalPrint.call(IMain.scala:773)
	at scala.tools.nsc.interpreter.IMain$Request.loadAndRun(IMain.scala:1020)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:627)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:626)
	at scala.reflect.internal.util.ScalaClassLoader$class.asContext(ScalaClassLoader.scala:31)
	at scala.reflect.internal.util.AbstractFileClassLoader.asContext(AbstractFileClassLoader.scala:19)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest.loadAndRunReq(IMain.scala:626)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:558)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:554)
	at scala.tools.nsc.interpreter.ILoop.reallyInterpret$1(ILoop.scala:804)
	at scala.tools.nsc.interpreter.ILoop.interpretStartingWith(ILoop.scala:849)
	at scala.tools.nsc.interpreter.ILoop.command(ILoop.scala:707)
	at scala.tools.nsc.interpreter.ILoop.processLine(ILoop.scala:405)
	at scala.tools.nsc.interpreter.ILoop.loop(ILoop.scala:431)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply$mcZ$sp(ILoop.scala:920)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.reflect.internal.util.ScalaClassLoader$.savingContextLoader(ScalaClassLoader.scala:97)
	at scala.tools.nsc.interpreter.ILoop.process(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop.main(ILoop.scala:931)
	at xsbt.ConsoleInterface.run(ConsoleInterface.scala:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at sbt.compiler.AnalyzingCompiler.call(AnalyzingCompiler.scala:101)
	at sbt.compiler.AnalyzingCompiler.console(AnalyzingCompiler.scala:76)
	at sbt.Console.sbt$Console$$console0$1(Console.scala:22)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply$mcV$sp(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Logger$$anon$4.apply(Logger.scala:85)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 75 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:165)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:340)
	... 70 more
Caused by: java.nio.channels.OverlappingFileLockException
	at sun.nio.ch.SharedFileLockTable.checkList(FileLockTable.java:255)
	at sun.nio.ch.SharedFileLockTable.add(FileLockTable.java:152)
	at sun.nio.ch.FileChannelImpl.tryLock(FileChannelImpl.java:1075)
	at org.h2.store.fs.FileNio.tryLock(FilePathNio.java:121)
	at org.h2.mvstore.cache.FilePathCache$FileCache.tryLock(FilePathCache.java:152)
	at java.nio.channels.FileChannel.tryLock(FileChannel.java:1155)
	at org.h2.mvstore.FileStore.open(FileStore.java:162)
	... 71 more
04-02 14:40:50 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:354)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2785)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2386)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:208)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:282)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:264)
	at org.h2.tools.Shell.connect(Shell.java:371)
	at org.h2.tools.Shell.promptLoop(Shell.java:209)
	at org.h2.tools.Shell.runTool(Shell.java:150)
	at org.h2.tools.Shell.main(Shell.java:79)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 29 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:170)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:340)
	... 24 more
04-02 14:41:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "RUN[*] SCRIPT 'sql/users.sql'
 "; expected "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE, REPLACE, {"; SQL statement:
run script 'sql/users.sql'
 [42001-186]
04-02 14:41:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "RUNSCRIPT 'sql/users.sql'[*]
 "; expected "FROM"; SQL statement:
runscript 'sql/users.sql'
 [42001-186]
04-02 14:42:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * ADDRESS[*] "; SQL statement:
select * address [42000-186]
04-02 14:47:54 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:354)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2785)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2386)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:208)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:282)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:264)
	at org.h2.tools.Shell.connect(Shell.java:371)
	at org.h2.tools.Shell.promptLoop(Shell.java:209)
	at org.h2.tools.Shell.runTool(Shell.java:150)
	at org.h2.tools.Shell.main(Shell.java:79)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 29 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:170)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:340)
	... 24 more
04-02 14:47:57 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:354)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2785)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2386)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:208)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:282)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:264)
	at org.h2.tools.Shell.connect(Shell.java:371)
	at org.h2.tools.Shell.promptLoop(Shell.java:209)
	at org.h2.tools.Shell.runTool(Shell.java:150)
	at org.h2.tools.Shell.main(Shell.java:79)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 29 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:170)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:340)
	... 24 more
04-02 14:51:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "U.UID" not found; SQL statement:

      insert into lives
      values ( u.uid ,  aid )
     [42122-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:247)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:72)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:277)
	at org.apache.commons.dbcp.DelegatingConnection.prepareStatement(DelegatingConnection.java:281)
	at org.apache.commons.dbcp.PoolingDataSource$PoolGuardConnectionWrapper.prepareStatement(PoolingDataSource.java:313)
	at scalikejdbc.DBSession$class.createStatementExecutor(DBSession.scala:82)
	at scalikejdbc.DBSession$class.updateWithFilters(DBSession.scala:390)
	at scalikejdbc.ActiveSession.updateWithFilters(DBSession.scala:547)
	at scalikejdbc.DBSession$class.updateWithFilters(DBSession.scala:372)
	at scalikejdbc.ActiveSession.updateWithFilters(DBSession.scala:547)
	at scalikejdbc.SQLUpdate$$anonfun$6.apply(SQL.scala:492)
	at scalikejdbc.SQLUpdate$$anonfun$6.apply(SQL.scala:492)
	at scalikejdbc.DBConnection$class.autoCommit(DBConnection.scala:222)
	at scalikejdbc.DB.autoCommit(DB.scala:60)
	at scalikejdbc.DB$$anonfun$autoCommit$1.apply(DB.scala:215)
	at scalikejdbc.DB$$anonfun$autoCommit$1.apply(DB.scala:214)
	at scalikejdbc.LoanPattern$class.using(LoanPattern.scala:18)
	at scalikejdbc.DB$.using(DB.scala:138)
	at scalikejdbc.DB$.autoCommit(DB.scala:214)
	at scalikejdbc.SQLUpdate.apply(SQL.scala:492)
	at cs220.users.Address.lives(Address.scala:34)
	at $line18.$read$$iw$$iw$$iw$$iw$.<init>(<console>:14)
	at $line18.$read$$iw$$iw$$iw$$iw$.<clinit>(<console>)
	at $line18.$eval$.$print$lzycompute(<console>:7)
	at $line18.$eval$.$print(<console>:6)
	at $line18.$eval.$print(<console>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at scala.tools.nsc.interpreter.IMain$ReadEvalPrint.call(IMain.scala:773)
	at scala.tools.nsc.interpreter.IMain$Request.loadAndRun(IMain.scala:1020)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:627)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:626)
	at scala.reflect.internal.util.ScalaClassLoader$class.asContext(ScalaClassLoader.scala:31)
	at scala.reflect.internal.util.AbstractFileClassLoader.asContext(AbstractFileClassLoader.scala:19)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest.loadAndRunReq(IMain.scala:626)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:558)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:554)
	at scala.tools.nsc.interpreter.ILoop.reallyInterpret$1(ILoop.scala:804)
	at scala.tools.nsc.interpreter.ILoop.interpretStartingWith(ILoop.scala:849)
	at scala.tools.nsc.interpreter.ILoop.command(ILoop.scala:707)
	at scala.tools.nsc.interpreter.ILoop.processLine(ILoop.scala:405)
	at scala.tools.nsc.interpreter.ILoop.loop(ILoop.scala:431)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply$mcZ$sp(ILoop.scala:920)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.reflect.internal.util.ScalaClassLoader$.savingContextLoader(ScalaClassLoader.scala:97)
	at scala.tools.nsc.interpreter.ILoop.process(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop.main(ILoop.scala:931)
	at xsbt.ConsoleInterface.run(ConsoleInterface.scala:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at sbt.compiler.AnalyzingCompiler.call(AnalyzingCompiler.scala:101)
	at sbt.compiler.AnalyzingCompiler.console(AnalyzingCompiler.scala:76)
	at sbt.Console.sbt$Console$$console0$1(Console.scala:22)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply$mcV$sp(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Logger$$anon$4.apply(Logger.scala:85)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:745)
04-02 15:35:12 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:354)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2785)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2386)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:75)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1148)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at scalikejdbc.CommonsConnectionPool.borrow(CommonsConnectionPool.scala:43)
	at scalikejdbc.DB$.readOnly(DB.scala:172)
	at scalikejdbc.SQLToResult$class.apply(SQL.scala:539)
	at scalikejdbc.SQLToOptionImpl.apply(SQL.scala:697)
	at cs220.users.User$.get(User.scala:81)
	at $line4.$read$$iw$$iw$$iw$$iw$.<init>(<console>:11)
	at $line4.$read$$iw$$iw$$iw$$iw$.<clinit>(<console>)
	at $line4.$eval$.$print$lzycompute(<console>:7)
	at $line4.$eval$.$print(<console>:6)
	at $line4.$eval.$print(<console>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at scala.tools.nsc.interpreter.IMain$ReadEvalPrint.call(IMain.scala:773)
	at scala.tools.nsc.interpreter.IMain$Request.loadAndRun(IMain.scala:1020)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:627)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest$$anonfun$loadAndRunReq$1.apply(IMain.scala:626)
	at scala.reflect.internal.util.ScalaClassLoader$class.asContext(ScalaClassLoader.scala:31)
	at scala.reflect.internal.util.AbstractFileClassLoader.asContext(AbstractFileClassLoader.scala:19)
	at scala.tools.nsc.interpreter.IMain$WrappedRequest.loadAndRunReq(IMain.scala:626)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:558)
	at scala.tools.nsc.interpreter.IMain.interpret(IMain.scala:554)
	at scala.tools.nsc.interpreter.ILoop.reallyInterpret$1(ILoop.scala:804)
	at scala.tools.nsc.interpreter.ILoop.interpretStartingWith(ILoop.scala:849)
	at scala.tools.nsc.interpreter.ILoop.command(ILoop.scala:707)
	at scala.tools.nsc.interpreter.ILoop.processLine(ILoop.scala:405)
	at scala.tools.nsc.interpreter.ILoop.loop(ILoop.scala:431)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply$mcZ$sp(ILoop.scala:920)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop$$anonfun$process$1.apply(ILoop.scala:906)
	at scala.reflect.internal.util.ScalaClassLoader$.savingContextLoader(ScalaClassLoader.scala:97)
	at scala.tools.nsc.interpreter.ILoop.process(ILoop.scala:906)
	at scala.tools.nsc.interpreter.ILoop.main(ILoop.scala:931)
	at xsbt.ConsoleInterface.run(ConsoleInterface.scala:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at sbt.compiler.AnalyzingCompiler.call(AnalyzingCompiler.scala:101)
	at sbt.compiler.AnalyzingCompiler.console(AnalyzingCompiler.scala:76)
	at sbt.Console.sbt$Console$$console0$1(Console.scala:22)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply$mcV$sp(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Console$$anonfun$apply$2$$anonfun$apply$1.apply(Console.scala:23)
	at sbt.Logger$$anon$4.apply(Logger.scala:85)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]" [50000-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 75 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/tim/vboxes/vagrant-cs220-linux-box/week-10-databases/code/database/users.mv.db [1.4.186/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:165)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:340)
	... 70 more
Caused by: java.nio.channels.OverlappingFileLockException
	at sun.nio.ch.SharedFileLockTable.checkList(FileLockTable.java:255)
	at sun.nio.ch.SharedFileLockTable.add(FileLockTable.java:152)
	at sun.nio.ch.FileChannelImpl.tryLock(FileChannelImpl.java:1075)
	at org.h2.store.fs.FileNio.tryLock(FilePathNio.java:121)
	at org.h2.mvstore.cache.FilePathCache$FileCache.tryLock(FilePathCache.java:152)
	at java.nio.channels.FileChannel.tryLock(FileChannel.java:1155)
	at org.h2.mvstore.FileStore.open(FileStore.java:162)
	... 71 more
04-02 16:18:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Function "AVERAGE" not found; SQL statement:
select average(age) from users [90022-186]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.readJavaFunction(Parser.java:2341)
	at org.h2.command.Parser.readFunction(Parser.java:2393)
	at org.h2.command.Parser.readTerm(Parser.java:2727)
	at org.h2.command.Parser.readFactor(Parser.java:2259)
	at org.h2.command.Parser.readSum(Parser.java:2246)
	at org.h2.command.Parser.readConcat(Parser.java:2216)
	at org.h2.command.Parser.readCondition(Parser.java:2066)
	at org.h2.command.Parser.readAnd(Parser.java:2038)
	at org.h2.command.Parser.readExpression(Parser.java:2030)
	at org.h2.command.Parser.parseSelectSimpleSelectPart(Parser.java:1942)
	at org.h2.command.Parser.parseSelectSimple(Parser.java:1974)
	at org.h2.command.Parser.parseSelectSub(Parser.java:1868)
	at org.h2.command.Parser.parseSelectUnion(Parser.java:1689)
	at org.h2.command.Parser.parseSelect(Parser.java:1677)
	at org.h2.command.Parser.parsePrepared(Parser.java:433)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.tools.Shell.execute(Shell.java:454)
	at org.h2.tools.Shell.promptLoop(Shell.java:293)
	at org.h2.tools.Shell.runTool(Shell.java:150)
	at org.h2.tools.Shell.main(Shell.java:79)
